### PR TITLE
Uninstall react-redux and axios

### LINKED ISSUE #6316: 🛠️ Replace `useDispatch` with `useQuery`/`request`
> ❕ Do not assign this EPIC to yourself. Sub-issues specified below can be assigned to yourself.

Currently, we are using `useDispatch` to dispatch actions through `fireRequest`. We need to migrate this to use `useQuery` (for performing GET requests on component mount) and `request` to perform API calls of other methods (such as form submissions, etc.)

### 📖 References for the `request` and `useQuery` data fetching utilities
- 📖 Documentation https://github.com/coronasafe/care_fe/blob/develop/src/Utils/request/README.md
- 👀 Example PR: #6327

### Tasks

_Feel free to split these individual tasks further into multiple sub-tasks/PRs if the refactor is large. It will be easier to do the refactor, QA testing, and code review._

- [x] #6330
- [x] #6326
- [x] #6331
- [x] #6328
- [x] #6360
- [x] #6372
- [x] #6376
- [x] #6391
- [x] #7036 
- [x] #6381 
- [x] #6442
- [x] #6392
- [x] #6546
- [x] #6443
- [x] #6483
- [x] #6530
- [x] #6705
- [x] #6888
- [x] #7168
- [x] #7169 
- [x] #7170 
- [x] #7171

### Steps to follow

1. Perform a global search with `from "react-redux"` and specify the corresponding files to include regex as shown

image

2. In all of the files in the search results, you'll have to remove the usage of `useDispatch` (and corresponding actions that are dispatched) and replace it with `useQuery` and `request` appropriately.

3. All of the existing routes are available at [`src/Redux/api.tsx`](https://github.com/coronasafe/care_fe/blob/develop/src/Redux/api.tsx) which can be passed to the `useQuery/request` directly.

4. For each route that is being used, ensure the response type is also present. If not, you'll have to add it.

Example:

```tsx
config: {
  path: "/config.json",
  noAuth: true,
  TRes: Res<IConfig>(),  // 👈 Add this
},
```

5. Delete the relevant no longer used actions from `actions.tsx`. (Ensure the action is not used outside the scope of the migration too)

6. You'll also need to add cypress tests if existing cypress tests do not cover pages/components where the replacements are being done.

### LINKED ISSUE #6376: 🛠️ Replace `useDispatch` w. `useQuery`/`request`: Facility (Part 1, A-D) (`src/Components/Facility/*`)

### PR DESCRIPTION
**Pull request by rithviknishad**
## Proposed Changes

- Fixes #6376 
- Fixes #6316

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by khavinshankar** [comment]
@rithviknishad can you please fix the failing cypress tests

File: src/index.tsx [hunk 1]
```diff
@@ -1,10 +1,6 @@
 import { createRoot } from "react-dom/client";
-import reducer from "../src/Redux/Reducer";
 import App from "./App";
 import "./i18n";
-import { applyMiddleware, createStore } from "redux";
-import thunk from "redux-thunk";
-import { Provider } from "react-redux";
```
Comment (sainak) [inline src/index.tsx:7]: It should be removed from package-lock as well

**Comment by bodhish** [comment]
Wohoo, atlast its happening 🚀

**Comment by rithviknishad** [comment]
Note: Requires https://github.com/ohcnetwork/care_fe/pull/8253 before this is merged

**Comment by rithviknishad** [comment]
Will resolve conflicts one's base PR is merged

**Comment by nihal467** [comment]
LGTM

**Comment by bodhish** [comment]
@rithviknishad lets get this merged asap

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**